apis/rds/v1alpha1/custom_types.go

```diff
@@ -49,8 +58,9 @@ type CustomDBClusterParameters struct {
 	// The password for the master database user. This password can contain any
 	// printable ASCII character except "/", """, or "@".
 	//
-	// Constraints: Must contain from 8 to 41 characters. Required.
-	MasterUserPasswordSecretRef xpv1.SecretKeySelector `json:"masterUserPasswordSecretRef"`
+	// Constraints: Must contain from 8 to 41 characters.
+	// +optional
+	MasterUserPasswordSecretRef *xpv1.SecretKeySelector `json:"masterUserPasswordSecretRef"`
```

Comment:
Does AWS API allow creating `DBCluster` without a password?

Reply:
Yeah, it's optional.